src/main/java/com/rcv/GuiConfigController.java

```diff
@@ -243,7 +252,7 @@ public void buttonDeleteCandidateClicked() {
 
   @Override
   public void initialize(URL location, ResourceBundle resources) {
-    Logger.executionLog(Level.INFO, "Opening config creator GUI...");
+    Logger.executionLog(Level.FINE, "Opening config creator GUI...");
```

Comment:
What's the difference?

Reply:
FINE isn't logged in the GuiLogger, so there's less noise in there.